fitbook1/src/index.css

```diff
@@ -15,9 +15,17 @@ body {
   --primary-text-color: #323232;
   --secondary-text-color: #ffffff;
   --primary-color: #2d3092;
+  --dark-primary-color: #272a7d; 
   --background-color: #f7fafb;
+  --background-color-dark:#e9ebec;
   --color-black: #000000;
   --color-white: #ffffff;
+  --color-yellow: #ffc83d;
+  --color-gray: #808080;
+
+  --z-index-highest: 1000;
```

Comment:
why use such high z index values?

Reply:
I had to define 3 z-index values so i just took these values